### DISCUSSION TITLE
Ignore webhook param on Restful API

### DISCUSSION THREAD

**Discussion by dokime7** [directus/directus#10975] [Feature Requests]
Hi,

It would be great to have a global parameter on the API to bypass webhook execution (created on admin settings/webhooks).

In my case I setup a webhook on web admin for calling an n8n workflow on update a collection. The n8n workflow update some rows on the directus collection, and....... infinite looping !

So, webhooks prevent execution should be useful.

If there is another solution to achieve that I will take it.

Thanks

**Comment by rijkvanzanten** [comment]
Great idea! I think it's also something we can make configurable on the webhook side itself (rather than relying on a query param to explicitly skip the execution—which could also be used by malicious users) 🤔

**Reply by benhaynes** [reply]
_Project Waterway_ 🤐

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
